notebooks/samples/explanations/tf2/ai-explanations-image.ipynb

```diff
@@ -95,7 +95,7 @@
       "source": [
```

Comment:
<p>It looks like there's 4 lines of empty space in <code>get_validation_dataset()</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>